ww_queries.js

```diff
@@ -119,8 +123,12 @@ async function startGame(maxPlayers) {
        or gpl_leader)`,
       [game.gms_id]
     );
+    const [rows2] = await promisePool.query(
```

Comment:
promisePool.query geeft rows als een named variable in een array terug, rows2 is daardoor niets hier.